### DISCUSSION TITLE
Yieldly's proposal for the establishment of ASASTATS-ASASTATS distribution and Tinyman's ASASTATS-YLDY liquidity pools

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#354] [General]
Emily from Yieldly reached us in a dedicated Telegram group with a proposal for two new pools on their platform:

- - - - -

To maximise exposure over the next 6 months - 180 day pools. Designed to payout over that time no sooner. These are not aimed at existing community, but rather bringing in new folk.

Suggested setup:

50m-75m ASAStats~ over 6 months (180 days) 

ASAstats > ASAstats 

YLDY/ASAStats LP > YLDY - 5m YLDY (provided by Yieldly) - 90 days

- - - - -

I have explained that a public discussion in our GitHub will define if we are interested or not in such an arrangement. 

Just a note that we haven't got such an arrangement so far. In the first establishment, we had all three types of their pools, in the second we didn't have this distribution pool, while in this proposal there's no staking pool.

**Comment by AlgoRhythMatic** [comment]
While I was a huge proponent of Yieldly at their launch, I have not been so inspired by much that has happened in their project space since that time (pretty dead over the past 6 months or more).  Plainly stated, I don't think Yieldly offers anything interesting that we are not able to obtain elsewhere.

**Reply by marucat27** [reply]
Since we already committed to Cometa staking, I'd say we stick to Cometa currently and when that period ends can decide if we move to another platform.  Yieldly is a more popular platform but having two different staking platforms I feel will make it difficult for users to decide where to stake.  I don't think there are other staking solutions currently on Algorand except for Algofi but I don't think that would give us much more exposure.

**Reply by marucat27** [reply]
I'm not sure if this is possible but what if Yieldly made a staking widget for ASA Stats Mobile App? ASA Stats probably has the most exposure of all sites I imagine.  A staking solution widget to stake directly via ASA Stats would be huge and if this could be done all within the Mobile App, that would be something to consider.  Just thoughts.

**Reply by ipaleka** [reply]
Yes, feel free to add it under the dependable user widgets wiki, together with that for Cometa you mentioned in Discord.

https://github.com/asastats/channel/wiki/FeaturesDependentOnUserWidgets

**Reply by ipaleka** [reply]
I guess the words of First Writer and First Community Manager are enough to conclude that we're not ready to participate based on this proposal.

Also, @AlgoRhythMatic you were talking [about the problem Yieldly has with their distribution pools](https://github.com/asastats/channel/discussions/60#discussioncomment-2815537), does that affects our future plans (as Maru suggests) with Yieldly distribution pools?

**Reply by AlgoRhythMatic** [reply]
> does that affects our future plans (as Maru suggests) with Yieldly distribution pools?

I think we should of course always keep an open mind when looking towards the future, so I’d prefer not to rule anything out.

For now, however, I don’t see that Yieldly offers any advantages that are more compelling than what we receive from Cometa for the near term (for staking or distribution).

**Reply by marucat27** [reply]
> > does that affects our future plans (as Maru suggests) with Yieldly distribution pools?
> 
> 
> 
> I think we should of course always keep an open mind when looking towards the future, so I’d prefer not to rule anything out.
> 
> 
> 
> For now, however, I don’t see that Yieldly offers any advantages that are more compelling than what we receive from Cometa for the near term (for staking or distribution).

I agree 👍
